test/shuffling_test.py

```diff
@@ -48,6 +49,64 @@ def test_seed(self):
         new = np.load("Be_REshuffled1.out.npy")
         assert np.isclose(np.sum(np.abs(old-new)), 0.0, atol=accuracy)
 
+    def test_seed_openpmd(self):
+        """Test that the shuffling is handled correctly internally."""
```

Comment:
If I understand this correctly, this test makes sure that Numpy- and openPMD-shuffling give the same results?
In that case, I would maybe make the documentation a bit more concrete here.

Reply:
Makes sense. I fixed that.